normalize-own-data/README.md

```diff
@@ -11,4 +40,10 @@ If you are interested in comparing your own data to data you have downloaded fro
 
 You can read more about the reference distribution and how it is generated in the [quantile normalization section](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization) of our [documentation](http://docs.refine.bio/en/latest/).
 
-In this [example notebook](https://github.com/AlexsLemonade/refinebio-examples/blob/master/normalize-own-data/quantile_normalize_own_data.Rmd), we demonstrate how to obtain the reference distribution from the refine.bio API and apply it to your own data in R.
+In this [example notebook](https://alexslemonade.github.io/refinebio-examples/normalize-own-data/quantile_normalize_own_data.nb.html), we demonstrate how to obtain the reference distribution from the refine.bio API and apply it to your own data in R.
```

Comment:
Normal people might not know what an API is. But also they can probably just fill in the gaps. *shrug*

Reply:
This is generally intended for a different audience than the differential expression module for example.

Reply:
Ah. That's true.